docs/source/index.rst

```diff
@@ -65,6 +53,21 @@ such topics as:
    R <https://arrow.apache.org/docs/r/>
    Ruby <https://github.com/apache/arrow/blob/master/ruby/README.md>
    Rust <https://docs.rs/crate/arrow/>
+   status
```

Comment:
I'm not sure, but it seems better to me to put forward the status matrix, since it gives a synthetic view of what each library (or "supported environment") supports.

Reply:
FYI, the reason why I moved it at the end is that the primary thing readers are probably going to look for is their own target language. So the Implementation comparison makes sense as an additional information but probably it's not the first thing you are looking for, you probably want to see "Python"/R so that you can quickly jump into the doc you are looking for. The comparison between platforms is probably going to come in a second time when you ask yourself why you can't do something on X while you could on Y or situations like that.

I don't have a strong opinion about this one, but I think it's helpful to keep it out of the list of supported platforms and moving it at the end was a quick way to get it out of the way of most readers apart from those explicitly looking for it.

Reply:
This sounds fair to me :-)